jsapp/scss/components/_kobo.map.scss

```diff
@@ -399,6 +399,28 @@ body .leaflet-control-layers,
   }
 }
 
+.map-no-geopoint{
+  text-align:center;
+  margin:auto;
+  padding:10px;
+  font-weight:bold;
+}
+.map-no-geopoint-wrapper{
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Here you can drop `left` and change `transform` to `transform: translateY(-50%);` - the text is already centered horizontally using `text-align: center`